### ISSUE TITLE
maturity and comparison with azure-storage-cpp

### ISSUE THREAD

**Issue by asafka-zerto** [Azure/azure-storage-cpplite#76]
Hello,

We're considering usage of this library (azure-storage-cpp) for a release that is expected to be G.A. in about 6 months from today. We need to access azure blob storage from a C++ service on Linux servers. We need to understand:

1. Is this library ready for production? is it being used in production by anyone? we're concerned by the "0.3.0" release tag, which might indicate lack of maturity for production.
2. Some comparison with azure-storage-cpp. Based on our tests this library is indeed lighter in terms of memory consumption and number of threads, which are quite important to us, but we'd love a more comprehensive comparison made by anyone.

**Comment by JinmingHu-MSFT** [comment]
> 2\. Some comparison with azure-storage-cpp. Based on our tests this library is indeed lighter in terms of memory consumption and number of threads, which are quite important to us, but we'd love a more comprehensive comparison made by anyone.

Yes, cpplite is designed to be lightweight and have less dependencies. Our previous test results showed that cpplite has better performance over cpp sdk. But currently cpplite only support those essential APIs while cpp supports a full set of APIs.

**Comment by katmsft** [comment]
Hi, we will continuously follow up on this conversation with the on-going email thread. Thanks for raising this.

**Comment by bhardwajs** [comment]
@katmsft - how can we follow up on here? I am in the same situation as @asafka-zerto 

Happy to follow-up in MSFT internal threads. Thanks!
